### PR TITLE
docs: improve formatting consistency

### PR DESCRIPTION
**Pull request by petarvujovic98**
Continuing work where I left of in #8094

### CODE REVIEW

**Comment by akhi3030** [comment]
@petarvujovic98: thanks for the PR, are you still working on this or is it ready for a review now?

**Comment by petarvujovic98** [comment]
Hey @akhi3030. I am still working on it, I will let you know when I get through all of the doc files.

**Comment by akhi3030** [comment]
Got it.  I'll mark it as draft for now then.  Please mark it as ready for review when you are ready.

**Comment by petarvujovic98** [comment]
Will do. Sorry for not marking it as a draft earlier.

**Comment by petarvujovic98** [comment]
@akhi3030 This is ready for review.

**Review by nikurt — APPROVED** [review]
Fantastic. Thank you for bringing consistency and proof-reading the documentation.

File: docs/architecture/how/proofs.md [hunk 1]
```diff
@@ -44,17 +43,16 @@ are:
...
 
-
 ```
 # information needed to verify that node v[1] is present in a tree
 # with a given root (n7)
 [(Left, n0), (Right, n6)]
 
 # Verification
-assert_eq(root, hash(hash(n0, hash(v[1])),n6)))
+assert_eq(root, hash(hash(n0, hash(v[1])), n6))
```
[the remark this answers is not in the document]
Comment (petarvujovic98) [reply docs/architecture/how/proofs.md:52]: It should, I'll update.

File: docs/architecture/how/proofs.md [hunk 2]
```diff
@@ -86,13 +84,14 @@ ShardChunkHeaderV3 {
...
     ),
+}
 ```
 
 The field that we care about is called `outcome_root`. This value represents the
 root of the binary merkle tree, that is created based on all the receipts that
 were processed in this chunk.
 
-**Note:** You can notice that we also have a field here called
+**Note**: You can notice that we also have a field here called
```
[the remark this answers is not in the document]
Comment (petarvujovic98) [reply docs/architecture/how/proofs.md:94]: No problem, I'll change the formatting.

**Comment by akhi3030** [comment]
@petarvujovic98: let me know if this is ready to be merged then.

**Comment by petarvujovic98** [comment]
@akhi3030 Ready for merging.

**Pull request merged**